### PR TITLE
F#src src 602 update yaml fields

### PR DESCRIPTION
**Pull request by dg-shadow**

### CODE REVIEW

File: sr_ethercat_hand_config/controls/motors/lh/motor_board_effort_controllers.yaml [hunk 1]
```diff
@@ -1,340 +1,380 @@
...
-    imax: 550
+    imax: 0
     max_pwm: 600
     p: 6200
     sg_left: 0
     sg_right: 0
     sgleftref: 0
     sgrightref: 0
     sign: 0
+    torque_limit: 500
```
Comment (toliver) [inline sr_ethercat_hand_config/controls/motors/lh/motor_board_effort_controllers.yaml:73]: Please check with @jlaforga but I think the torque limit should be the same as in the old imax (550 in this case).

Not sure why we reduced it in our June delivery.

**Review by toliver — CHANGES_REQUESTED** [review]

File: sr_ethercat_hand_config/controls/motors/lh/motor_board_effort_controllers.yaml (hunk 1 above)
Comment (dg-shadow) [reply sr_ethercat_hand_config/controls/motors/lh/motor_board_effort_controllers.yaml:73]: I copied the values from right->left, so this means the old left hand had a value different to the right. I suspect they should be the same. @jlaforga - can you answer this?

File: sr_ethercat_hand_config/controls/motors/rh/motor_board_effort_controllers.yaml [hunk 2]
```diff
@@ -1,340 +1,380 @@
...
-    imax: 550
+    imax: 0
     max_pwm: 600
     p: 6200
     sg_left: 0
     sg_right: 0
     sgleftref: 0
     sgrightref: 0
     sign: 0
+    torque_limit: 500
```
Comment (toliver) [inline sr_ethercat_hand_config/controls/motors/rh/motor_board_effort_controllers.yaml:16]: But here it is also happening. 550 vs 500

File: sr_ethercat_hand_config/controls/motors/rh/motor_board_effort_controllers.yaml [hunk 3]
```diff
@@ -1,340 +1,380 @@
...
   pid:
-    d: 64
+    d: 0
     deadband: 5
     f: 300
     i: 0
-    imax: 350
-    max_pwm: 300
+    imax: 0
+    max_pwm: 400
```
Comment (toliver) [inline sr_ethercat_hand_config/controls/motors/rh/motor_board_effort_controllers.yaml:47]: And here max_pwm has changed.

File: sr_ethercat_hand_config/controls/motors/rh/motor_board_effort_controllers.yaml [hunk 4]
```diff
@@ -1,340 +1,380 @@
...
-    imax: 700
+    imax: 0
     max_pwm: 600
     p: 6200
     sg_left: 0
     sg_right: 0
     sgleftref: 0
     sgrightref: 0
     sign: 0
+    torque_limit: 450
```
Comment (toliver) [inline sr_ethercat_hand_config/controls/motors/rh/motor_board_effort_controllers.yaml:92]: Although to be honest 450 seems more sensible than 700.

File: sr_ethercat_hand_config/controls/motors/lh/motor_board_effort_controllers.yaml (hunk 1 above)
Comment (jlaforga) [reply sr_ethercat_hand_config/controls/motors/lh/motor_board_effort_controllers.yaml:73]: Hello,
These values are estimations that we've been using historically and we didn't have problems with, I provided them to Dan. The only thing I would like to change is the TH1  & TH2 to 400, I 've just remember that OpenAI had 450 and the T-bar broke, so let's going to reduce TH1 and TH2 torque_limit to 400. The rest of the values should be OK.
Cheers,

**Pull request merged**